packages/utils-compat-data/scripts/mdn-browser-compat-data.js

```diff
@@ -35,7 +35,7 @@ const isUniversalSupportStatement = (browserName, supportStatement) => {
         return true; // Count unknown as universal support.
     }
 
-    if (version === '1') {
+    if (version === '1' || version === '1.0') {
```

Comment:
This saved ~10KB minified and gzipped, making up for the addition of new data so I didn't have to bump our bundle sizes 🎉